tests/conftest.py

```diff
@@ -32,6 +32,7 @@ async def yws_server(request):
     try:
         async with websocket_server, serve(websocket_server.serve, "127.0.0.1", 1234):
             yield websocket_server
+            websocket_server.stop()
```

Comment:
What's the point of this, since WebsocketServer.__aexit__() does the same as WebsocketServer.stop()?

Reply:
I'm not sure. However, the tests failed when I ran them this morning and passed after I added that line. No clue why the error only surfaced just this morning during my manual testing and not in the pipeline.